tests/phpunit/tests/formatting/wpTrimExcerpt.php

```diff
@@ -92,4 +92,49 @@ public function test_should_generate_excerpt_for_empty_values() {
 		$this->assertSame( 'Post content', wp_trim_excerpt( null, $post ) );
 		$this->assertSame( 'Post content', wp_trim_excerpt( false, $post ) );
 	}
+
+	/**
+	 * @ticket 56588
+	 * @covers ::wp_trim_excerpt
+	 */
+	public function test_wp_trim_excerpt_avoids_parsing_content_tags() {
+		$post = self::factory()->post->create();
+
+		/*
+		 * Record that during 'the_content' filter run by wp_trim_excerpt() the
+		 * wp_filter_content_tags() callback is not used.
+		 */
+		$has_filter = true;
+		add_filter(
+			'the_content',
+			function( $content ) use ( &$has_filter ) {
```

Comment:
```suggestion
			static function( $content ) use ( &$has_filter ) {
```
Use static closures whenever $this is not used to avoid memory leaks.